rotkehlchen/types.py

```diff
@@ -646,7 +647,8 @@ class CostBasisMethod(SerializableEnumMixin):
     ACB = auto()
 
 
-class AddressbookEntry(NamedTuple):
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=True, frozen=False)
+class NamedAddressbookEntry():
```

Comment:
This dataclass is not supposed to be instantiated right? is there a way to mark it in addition to having it in the docstring? Not sure if just adding metaclass as parent class avoids that